core/controllers/outgoing_emails.py

```diff
@@ -19,83 +19,76 @@
 from core.controllers import base
 from core.domain import email_manager
 from core.utility import string_validator
-import config
-
-@staticmethod
-def write_email_contents(user_comment, user_organization):
-    """Compose email body with given user info.
-
-    Arguments:
-        user_comment: str. User's comment to be used as the primary email
-            content.
-        user_organization: str|None. User's organization name to be written at
-            the top of the email body.
-
-    Returns:
-        str. Email body to be sent.
-
-    Raises:
-        InvalidStringException: The provided string is invalid.
-    """
-    email_contents = ''
-    if user_organization is not None:
-        string_validator.check_string_valid(user_organization)
-        email_contents = ('Organization: %s\n' % user_organization)
-
-    string_validator.check_string_valid(user_comment)
-    email_contents += user_comment
-    return email_contents
-
 
-@staticmethod
-def write_email_subject(subject_type):
-    """Append user email to the email subject.
-
-    Arguments:
-        subject_type: str. The type of email subject to be returned.
-
-    Returns:
-        str. The subject of the email to be sent.
-
-    Raises:
-        InvalidStringException: The provided string is invalid.
-    """
-    string_validator.check_string_valid(subject_type)
-    subject_type = subject_type.upper()
-    email_subject = ''
-
-    if subject_type == config.PARTNERSHIPS_TYPE:
-        email_subject = 'Partnering with Oppia'
-    elif subject_type == config.VOLUNTEER_TYPE:
-        email_subject = 'Volunteering with Oppia'
-    elif subject_type == config.DEFAULT_TYPE:
-        email_subject = 'General question about Oppia Foundation'
-    else:
-        raise KeyError('Invalid subject type')
-    return email_subject
+import config
 
 
 class ForwardToAdminEmailHandler(base.BaseHandler):
     """Handler for forwarding email to admin email address after
         user submits Contact us form."""
 
+    @classmethod
+    def write_email_subject(cls, email_type):
+        """Append user email to the email subject.
+
+        Arguments:
+            email_type: str. The type of email subject to be returned.
+
+        Returns:
+            str. The subject of the email to be sent.
+
+        Raises:
+            InvalidStringException: The provided string is invalid.
+        """
+        string_validator.check_string_valid(email_type)
+        email_type = email_type.upper()
```

Comment:
There should be no transformations; pass the correct thing directly.

Reply:
Hmmm looks like some left-over from me trying to do invalidation within the POST method.

Removed!